<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
      "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <title>Publications</title>
  <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
  <meta name="copyright" content="webdesign@wdesign-it.com" />
  <meta name="Description" content="Criao da sua imagem on-line" />
  <meta name="Keywords"
  content="metabolomics bioinformatics"
  />
  <meta name="author" content="webdesign@wdesign-it.com" />
  <link href="file:///E|/clia/wdesign-it/oswd_projecto1/style_blue.css"
  rel="stylesheet" type="text/css" />
  <link href="style_blue3.css" rel="stylesheet" type="text/css" />
</head>

<body>

<div id="container">

<div id="banner">

<div class="navcontainer">
<ul id="navlist">
  <li><a href="index.html">Home</a></li>
  <li><a href="research.html">Research</a></li>
  <li><a href="publications.html">Publications</a></li>
  <li><a href="projects.html">Projects</a></li>
  <li><a href="team.html">Team</a></li>  
  <li><a href="training.html">Training</a></li>
  <li><a href="software.html">Software</a></li>
  <li><a href="jobs.html">Jobs</a></li>  
</ul>
</div>
</div>

<div id="containerboth">

<div id="sidebar-a">

</div>

<div id="content">
<h2>Articles</h2>

<ol>
  <li><p>Grassin-Delyle S., Roquencourt C., Moine P., Saffroy G., Carn S., Heming N., Fleuriet J., Salvator H., Naline E., Couderc L.-J., Devillier P., Thevenot E. A. and Annane D. (2021). Metabolomics of exhaled breath in critically ill COVID-19 patients: A pilot study. <em>EBioMedicine</em>, <strong>63</strong>, doi:<a href="https://doi.org/10.1016/j.ebiom.2020.103154">10.1016/j.ebiom.2020.103154</a></p></li>
  <li><p>Safi-Stibler S., Thevenot E. A., Jouneau L., Jouin M., Seyer A., Jammes H., Rousseau-Ralliard D., Baly C. and Gabory A. (2020). Differential effects of post-weaning diet and maternal obesity on mouse liver and brain metabolomes. <em>Nutrients</em>, <strong>12</strong>, doi:<a href="https://doi.org/10.3390/nu12061572">10.3390/nu12061572</a></p></li>
  <li><p>Fall F., Lamy E., Brollo M., Naline E., Lenuzza N., Thevenot E., Devillier P. and Grassin-Delyle S. (2020). Metabolic reprograming of LPS-stimulated human lung macrophages involves tryptophan metabolism and the aspartate-arginosuccinate shunt. <em>PLOS ONE</em>, <strong>15</strong>, doi:<a href="https://doi.org/10.1371/journal.pone.0230813">10.1371/journal.pone.0230813</a></p></li>
  <li><p>Stanstrup J., Broeckling C.D., Helmus R., Hoffmann N., Mathe E., Naake T., Nicolotti L., Peters K., Rainer J., Salek R., Schulze T., Schymanski E.L., Stravs M.A., Thevenot E.A., Treutler H., Weber R., Willighagen E., Witting M., Neumann S. The metaRbolomics toolbox in Bioconductor and beyond. <em>Metabolites</em>, <strong>9</strong>, doi:<a href="https://doi.org/10.3390/metabo9100200">10.3390/metabo9100200</a></p></li>
  <li><p>Fall F., Lenuzza N., Lamy E., Brollo M., Naline E., Devillier P., Thevenot E. and Grassin-Delyle S. (2019). A split-range acquisition method for the non-targeted metabolomic profiling of human plasma with hydrophilic interaction chromatography - high-resolution mass spectrometry. <em>Journal of Chromatography B</em>, <strong>1128</strong>, doi:<a href="https://doi.org/10.1016/j.jchromb.2019.121780">10.1016/j.jchromb.2019.121780</a></p></li>
  <li><p>Emami Khoonsari P., Moreno P., Bergmann S., Burman J., Capuccini M., Carone M., Cascante M., de Atauri P., Foguet C., Gonzalez-Beltran A., Hankemeier T., Haug K., He S., Herman S., Johnson D., Kale N., Larsson A., Neumann S., Peters K., Pireddu L., Rocca-Serra P., Roger P., Rueedi R., Ruttkies C., Sadawi N., Salek R.M., Sansone S.A., Schober D., Selivanov V., Thevenot E.A., van Vliet M., Zanetti G., Steinbeck C., Kultima K. and Spjuth O. (2019). Interoperable and scalable data analysis with microservices: applications in metabolomics. <em>Bioinformatics</em>, <strong>35</strong>:3752-3760, doi:<a href="https://doi.org/10.1093/bioinformatics/btz160">10.1093/bioinformatics/btz160</a></p></li>
  <li><p>Peters K., Bradbury J., Bergmann S., Capuccini M., Cascante M., de Atauri P., Ebbels T.M.D., Foguet C., Glen R., Gonzalez-Beltran A., Gunther U.L., Handakas E., Hankemeier T., Haug K., Herman S., Holub P., Izzo M., Jacob D., Johnson D., Jourdan F., Kale N., Karaman I., Khalili B., Emami Khonsari P., Kultima K., Lampa S., Larsson A., Ludwig C., Moreno P., Neumann S., Novella J.A., O'Donovan C., Pearce J.T.M., Peluso A., Piras M.E., Pireddu L., Reed M.A.C., Rocca-Serra P., Roger P., Rosato A., Rueedi R., Ruttkies C., Sadawi N., Salek R.M., Sansone S.A., Selivanov V., Spjuth O., Schober D., Thevenot E.A., Tomasoni M., van Rijswijk M., van Vliet M., Viant M.R., Weber R.J.M., Zanetti G. and Steinbeck C. (2019). PhenoMeNal: processing and analysis of metabolomics data in the cloud. <em>Gigascience</em>, <strong>8</strong>, doi:<a href="https://doi.org/10.1093/gigascience/giy149">10.1093/gigascience/giy149</a></p></li>
  <li><p>Souard F., Delporte C., Stoffelen P., Thevenot E.A., Noret N., Dauvergne B., Kauffmann J.-M., Van Antwerpen P. and Stevigny C. (2017). Metabolomics fingerprint of coffee species determined by untargeted-profiling study using LC-HRMS. <em>Food Chemistry</em>, <strong>245</strong>:603-612. doi:<a href="https://doi.org/10.1016/j.foodchem.2017.10.022">10.1016/j.foodchem.2017.10.022</a></p></li>
  <li><p>van Rijswijk M., Beirnaert C., Caron C., Cascante M., Dominguez V., Dunn W., Ebbels T., Giacomoni F., Gonzalez-Beltran A., Hankemeier T., Haug K., Izquierdo-Garcia J., Jimenez R., Jourdan F., Kale N., Klapa M., Kohlbacher O., Koort K., Kultima K., Le Corguille G., Moschonas N., Neumann S., O'Donovan C., Reczko M., Rocca-Serra P., Rosato A., Salek R., Sansone S., Satagopam V., Schober D., Shimmo R., Spicer R., Spjuth O., Thevenot E., Viant M., Weber R., Willighagen E., Zanetti G. and Steinbeck C. (2017). The future of metabolomics in ELIXIR. <em>F1000Research</em>, doi:<a href="https://doi.org/10.12688/f1000research.12342.1">10.12688/f1000research.12342.1</a>
  <li><p>Delabriere A., Hohenester U.M., Colsch B., Junot C., Fenaille F. and Thevenot E.A. (2017). <em>proFIA</em>: a data preprocessing workflow for flow injection analysis coupled to high-resolution mass spectrometry. <em>Bioinformatics</em>, <strong>33</strong>:3767-3775. doi:<a href="https://doi.org/10.1093/bioinformatics/btx458">10.1093/bioinformatics/btx458</a> <a href="https://hal.archives-ouvertes.fr/hal-01574347">[HAL-pdf]</a></p></li>
  <li><p>Guitton Y., Tremblay-Franco M., Le Corguille G., Martin J.-F., Petera M., Roger-Mele P., Delabriere A., Goulitquer S., Monsoor M., Duperier C., Canlet C., Servien R., Tardivel P., Caron C., Giacomoni F. and Thevenot, E.A. (2017). Create, run, share, publish, and reference your LC-MS, FIA-MS, GC-MS, and NMR data analysis workflows with the Workflow4Metabolomics 3.0 Galaxy online infrastructure for metabolomics. <em>International Journal of Biochemistry and Cell Biology</em>, <strong>93</strong>:89-101. doi:<a href="https://doi.org/10.1016/j.biocel.2017.07.002">10.1016/j.biocel.2017.07.002</a> <a href="https://hal.archives-ouvertes.fr/hal-01574351">[HAL-pdf]</a></p></li>
  <li><p>Rinaudo P., Boudah S., Junot C. and Thevenot
    E.A. (2016). <em>biosigner</em>: a new method for the discovery of
    significant molecular signatures from omics data. <em>Frontiers in
    Molecular Biosciences</em>, <strong>3</strong>. doi:<a href="https://doi.org/10.3389/fmolb.2016.00026">10.3389/fmolb.2016.00026</a></p></li>
  <li><p>Thevenot E.A., Roux A., Xu Y., Ezan E. and Junot C. (2015). Analysis of
    the human adult urinary metabolome variations with age, body mass index and
    gender by implementing a comprehensive workflow for univariate and OPLS
    statistical analyses. <em>Journal of Proteome Research</em>, <strong>14</strong>:3322-3335. doi:<a
    href="https://doi.org/10.1021/acs.jproteome.5b00354">10.1021/acs.jproteome.5b00354</a></p></li>
  <li><p>Roux A., Thevenot E., Seguin F., Olivier M.-F. and Junot C. (2015).
    Impact of collection conditions on the metabolite content of human urine
    samples as analyzed by liquid chromatography coupled to mass spectrometry
    and nuclear magnetic resonance spectroscopy. <em>Metabolomics</em>,
    <strong>11</strong>:1095:1105. doi:<a
    href="https://doi.org/10.1007/s11306-014-0764-5">10.1007/s11306-014-0764-5</a></p></li>
  <li><p>Giacomoni F., Le Corguille G., Monsoor M., Landi M., Pericard P., Petera
    M., Duperier C., Tremblay-Franco M., Martin J.-F., Jacob D., Goulitquer S.,
    Thevenot E.A. and Caron C. (2015). Workflow4Metabolomics: A collaborative
    research infrastructure for computational metabolomics.
    <em>Bioinformatics</em>, <strong>31</strong>:1493-1495. doi:<a
    href="https://doi.org/10.1093/bioinformatics/btu813">10.1093/bioinformatics/btu813</a></p></li>
  <li><p>Lenuzza N., Duval X., Nicolas G., Thevenot E., Job S., Videau O., Narjoz
    C., Loriot M.-A., Beaune P., Becquemont L., Mentre F., Funck-Brentano C.,
    Alavoine L., Arnaud P., Delaforge M. and Benech H. (2014). Safety and
    pharmacokinetics of the CIME combination of drugs and their metabolites
    after a single oral dosing in healthy volunteers. <em>European Journal of
    Drug Metabolism and Pharmacokinetics</em>, 1-14. doi:<a
    href="https://doi.org/10.1007/s13318-014-0239-0">10.1007/s13318-014-0239-0</a></p></li>
  <li><p>Lacombe O., Videau O., Chevillon D., Guyot A.-C., Contreras C., Blondel
    S., Nicolas L., Ghettas A., Benech H., Thevenot E., Pruvost A., Bolze S.,
    Krzaczkowski L., Prevost C. and Mabondzo A. (2011). In vitro primary human
    and animal cell-based blood-brain barrier models as a screening tool in
    drug discovery. <em>Molecular Pharmaceutics</em>,
    <strong>8</strong>:651-663. doi:<a
    href="https://doi.org/10.1021/mp1004614">10.1021/mp1004614</a></p></li>
  <li><p>Videau O., Pitarque S., Troncale S., Hery P., Thevenot E., Delaforge M.
    and Benech H. (2011). Can a cocktail designed for phenotyping
    pharmacokinetics and metabolism enzymes in human can be used efficiently in
    rats. <em>Xenobiotica</em>, <strong>42</strong>:349-354. doi:<a
    href="https://doi.org/10.3109/00498254.2011.625453">10.3109/00498254.2011.625453</a></p></li>
  <li><p>Videau O., Delaforge M., Levi M., Thevenot E., Gal O., Becquemont L.,
    Beaune P., Lirsac P., Grassi J. and Benech H. (2010). Biochemical and
    analytical developments of the CIME cocktail for drug fate assessment in
    humans. <em>Rapid Communications in Mass
    Spectrometry</em>,<strong>24</strong>:2407-2419. doi:<a
    href="https://doi.org/10.1002/rcm.4641">10.1002/rcm.4641</a></p></li>
  <li><p>Cote F., Thevenot E., Fligny C., Fromes Y., Darmon M., Ripoche M.A.,
    Bayard E., Hanoun N., Saurini F., Lechat P., Dandolo L., Hamon M., Mallet
    J. and Vodjdani G. (2003). Disruption of the nonneuronal tph1 gene
    demonstrates the importance of peripheral serotonin in cardiac function.
    <em>Proceedings of the National Academy of Sciences USA</em>,
    <strong>100</strong>:13525-30. doi:<a
    href="https://doi.org/10.1073/pnas.2233056100">10.1073/pnas.2233056100</a></p></li>
  <li><p>Thevenot E., Cote F., Colin P., He Y., Leblois H., Perricaudet M., Mallet
    J. and Vodjdani G. (2003). Targeting conditional gene modification into the
    serotonin neurons of the dorsal raphe nucleus by viral delivery of the Cre
    recombinase. <em>Molecular and Cellular
    Neuroscience</em>,<strong>24</strong>:139-47. doi:<a
    href="https://doi.org/10.1016/S1044-7431(03)00131-3">10.1016/S1044-7431(03)00131-3</a></p></li>
  <li><p>Cote F., Schussler N., Boularand S., Peirotes A., Thevenot E., Mallet J.
    and Vodjdani G. (2002). Involvement of NF-Y and Sp1 in basal and
    cAMP-stimulated transcriptional activation of the tryptophan hydroxylase
    (TPH) gene in the pineal gland. <em>Journal of
    Neurochemistry</em>,<strong>81</strong>:673-85. doi:<a
    href="https://doi.org/10.1046/j.1471-4159.2002.00890.x">10.1046/j.1471-4159.2002.00890.x</a></p></li>
  <li><p>De Gois S., Houhou L., Oda Y., Corbex M., Pajak F., Thevenot E., Vodjdani
    G., Mallet J. and Berrard S. (2000). Is RE1/NRSE a common cis-regulatory
    sequence for ChAT and VAChT genes? <em>Journal of Biological
    Chemistry</em>, <strong>275</strong>:36683-90. doi:<a
    href="https://doi.org/10.1074/jbc.M006895200">10.1074/jbc.M006895200</a></p></li>
</ol>

<h2>Book chapter</h2>

<ol>
  <li>Viral vectors for in vivo gene transfer. (2009). Thevenot E., Dufour N. and Deglon N. In Nanoscience: Nanotechnology and Nanobiology (Lahmani M., Boisseau P., Houdy P. eds.), Springer (Original French edition by Belin, 2007). doi:<a
    href="https://doi.org/10.1007/978-3-540-88633-4_23">10.1007/978-3-540-88633-4\_23</a></li>
</ol>

<h2>Newsletters</h2>

<ol>
  <li>Workflow4Metabolomics 2.0: New workflows for LC-HRMS, GC-MS, and NMR data processing, statistical analysis, and annotation (<a
    href="http://www.metabonews.ca/Jun2015/MetaboNews_Jun2015.htm#spotlight">MetaboNews, Issue 46, June 2015</a>)</li>
  <li>MetaboHUB: The French infrastructure for metabolomics and fluxomics (<a
    href="http://www.metabonews.ca/Sep2014/MetaboNews_Sep2014.htm#MetaboInterviews">MetaboNews, Issue 37, Sept. 2014</a>)</li>
</ol>

<h2>Seminars</h2>

<ol>
  <li>Proteomics and metabolomics data integration: where do we stand? (2020), <a href="https://jipromet-2020.sciencesconf.org/">3rd workshop on Proteomics and Metabolomics Data Integration</a></li>
  <li>Data sciences for deep phenotyping and precision medicine (2019), <a href="http://univ-cotedazur.fr/events/msi-seminars">Center of Modeling, Simulation, and Interactions</a>, Nice.</li>
  <li>Data sciences for deep phenotyping and precision medicine (2019), <a href="https://wikis.univ-lille.fr/bilille/metabolo2019">bilille Bioinformatics Platform</a>, Lille.</li>
  <li>Statistical workflows for computational metabolomics (2016), <a
    href="http://metabolomics2016.org/program/workshops">'metaRbolomics: The R toolbox for Metabolomics' workshop session from the Metabolomics Society Conference</a>, Dublin.</li>
  <li>The Workflow4Metabolomics online infrastructure for users and developers (2016), <a
    href="http://metabolomics2016.org/program/workshops">'Computational workflows and workflow engines' workshop session from the Metabolomics Society Conference</a>, Dublin.</li>
  <li><em>biosigner</em>: a new method and module for signature discovery from omics data (2016). <a
    href="https://10-js-rfmf.sciencesconf.org/">RFMF</a>, Montpellier.</li>
  <li>Meeting the statisticians' and experimenters' needs for reproducible workflows with Bioconductor and Galaxy: the example of the <em>ropls</em> and <em>biosigner</em> package integration into the <em>Workflow4metabolomics</em> computational infrastructure (2015), <a
    href="https://sites.google.com/site/eurobioc2015/home">European Bioconductor Developers' Meeting</a>, Cambridge.</li>
  <li>Statistical methods for biomarker discovery (2015), <a
    href="http://selectbiosciences.com/conferences/index.aspx?conf=Metabo2015">Metabomeeting</a>, Cambridge.</li>
  <li>The Worfklow4metabolomics infrastructure: meeting the workflow challenge (2015). <a
    href="http://selectbiosciences.com/conferences/index.aspx?conf=Metabo2015">Metabomeeting</a>, Cambridge.</li>
  <li>Urine metabolomics for biomarker discovery: Data analysis strategies to study human cohorts (2015). <a
    href="http://www.stbc.org.tn/fr/jnbc">Clinical Biology Conference</a>, Hammamet.</li>
  <li>Signal processing and data analysis applied to the study of the physiological variations of the urinary metabolome (2015). <a
    href="http://www.dim-analytics.fr/seminaires/journee-thematique-chimiometrie-13/article/matinee-thematique-chimiometrie-le">Chemometrics workshop, DIM Analytics</a>, Paris.</li>
  <li>Biostatistics for biomarker discovery and phenotype prediction (2014). <a
    href="http://137.132.165.245/metabolomics2014/index.html">Merlion Metabolomics Workshop</a>, Singapore.</li>
  <li>Statistical approaches to study the physiological variability of the urinary metabolome (2014). <a
    href="https://colloque6.inra.fr/8_js_rfmf_lyon_2014">RFMF</a>, Lyon.</li>
</ol>

</div>
</div>

<div id="footer">
</div>

</div>
</body>
</html>
