<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
      "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <title>Research</title>
  <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
  <meta name="copyright" content="webdesign@wdesign-it.com" />
  <meta name="Description" content="Criao da sua imagem on-line" />
  <meta name="Keywords"
  content="workflow4metabolomics biostatistics multivariate"
  />
  <meta name="author" content="webdesign@wdesign-it.com" />
  <link href="file:///E|/clia/wdesign-it/oswd_projecto1/style_blue.css"
  rel="stylesheet" type="text/css" />
  <link href="style_blue3.css" rel="stylesheet" type="text/css" />
</head>

<body>

<div id="container">

<div id="banner">

<div class="navcontainer">
<ul id="navlist">
  <li><a href="index.html">Home</a></li>
  <li><a href="research.html">Research</a></li>
  <li><a href="publications.html">Publications</a></li>
  <li><a href="projects.html">Projects</a></li>
  <li><a href="team.html">Team</a></li>  
  <li><a href="training.html">Training</a></li>
  <li><a href="software.html">Software</a></li>
  <li><a href="jobs.html">Jobs</a></li>
</ul>
</div>
</div>

<div id="containerboth">

<div id="sidebar-a">

</div>

<div id="content">

<p>Our research has been focusing for more than 12 years on <strong>omics data sciences</strong> for <strong>systems biology and precision medicine</strong>, with a particular emphasis on metabolomics data sciences.</p>

<p><strong>Data sciences</strong> provide powerful approaches and algorithms (signal processing, data mining, machine learning, artificial intelligence) for the <strong>processing and analysis of high-dimensional data</strong>, such as omics datasets. <strong>Metabolomics</strong> (untargeted analysis of small molecules
involved in biochemical reactions) is of major interest for phenotype
characterization and biomarker discovery. <strong>High-resolution mass
spectrometry (HRMS)</strong> is a technology of choice for metabolomics (and also for proteomics), due to
its sensitivity and resolution.</p>

<p><strong>Metabolomics data science is challenging</strong> for (at least) three reasons. First, the
<strong>signal</strong> generated by mass spectrometry instruments is specific
(retention time dimension, heteroscedastic noise, analytical drift). Second, as
in other omics, the <strong>number of detected features</strong> is higher than
the number of samples, and many variables are correlated. Third,
<strong>structural</strong> characterization of the metabolites based on their
mass and fragmentation pattern is often only partial due to the chemical
diversity of the metabolome. Our team has been developping innovative methods and tools to tackle each of these challenges.</p>

<br style="margin-bottom:240px;"/>

<p style="text-align:center"><img alt="Metabolomics data sciences" src="images/omics-data-sciences.png"
width="600" height="450" /></p>
<!-- <p style="text-align:center"><img alt="Computational metabolomics" src="images/metabohub_illustration.jpg"
width="384" height="288" /></p> -->

<br style="margin-bottom:240px;"/>

<strong>Direct injection methods</strong> (such as Flow Injection Analysis) are of particular interest for high-throughput phenotyping. We therefore developed an innovative <strong>preprocessing workflow</strong> which takes as input the individual raw files and generates the samples by variables table of intensities (peak table). The steps include (i) <strong>peak detection and quantification</strong> within each file, (ii) <strong>peak alignment</strong> across samples to generate the peak table, and (iii) <strong>missing value imputation</strong>. In particular, new methods were required to optimize step (i), including noise estimation, modeling of the injection peak, and precise determination of each analyte peak borders. Application to several real data sets resulted in robust and accurate detection and quantification. <a href="https://doi.org/10.18129/B9.bioc.proFIA"><strong>proFIA</strong></a> is available as an R/Bioconductor package and as a <a href="https://toolshed.g2.bx.psu.edu/"><strong>Galaxy module</strong></a> (<a href="http://dx.doi.org/10.1093/bioinformatics/btx458"><strong>Delabrière et al, 2017</strong></a>).

<br style="margin-bottom:240px;"/>

<p style="text-align:center"><img alt="proFIA" src="images/profia.png"
width="600" height="450" /></p>

<br style="margin-bottom:240px;"/>

<p>We implemented the <strong>Orthogonal Partial Least-Squares
(OPLS)</strong> approach for regression and classification from <a
href="http://dx.doi.org/10.1002/cem.695">Trygg and Wold (2002)</a> as an R
package named <a href="http://bioconductor.org/packages/ropls"><strong>ropls</strong></a> (<a
href="http://dx.doi.org/10.1021/acs.jproteome.5b00354"><strong>Thévenot et al,
2015</strong></a>). OPLS algorithm is a variation of PLS and allows to model separately
the orthogonal variation (i.e. non-correlated to the response) and the
predictive variation (i.e. correlated to the response), and thus facilitates
model interpretation.</p>

<br style="margin-bottom:240px;"/>

<p style="text-align:center"><img alt="ropls" src="images/res_ropls.jpg"
width="384" height="384" /></p>

<br style="margin-bottom:240px;"/>

<p>Finally, we developed a new methodology for
<strong>feature selection</strong>, of the wrapper type, which assesses the
significance of the features for the model performance (<a
href="http://bioconductor.org/packages/biosigner"><strong>biosigner</strong></a> R package). The
wrapping of three classifiers (PLS-DA, Random Forest and Support Vector
Machine) with this methodology resulted in <strong>stable</strong> signatures of <strong>restricted
size</strong>, when applied to real metabolomics and transcriptomics datasets (<a
href="http://dx.doi.org/10.3389/fmolb.2016.00026"><strong>Rinaudo et al, 2016</strong></a>).</p>

<br style="margin-bottom:240px;"/>

<p style="text-align:center"><img alt="biosigner" src="images/res_fs_tier.jpg"
width="384" height="336" /></p>

<br style="margin-bottom:240px;"/>

<p>All algorithms are available both as <strong> R packages </strong> (<a
href="http://bioconductor.org">Bioconductor</a> repository) and <strong> Galaxy modules </strong> (<a href="http://workflow4metabolomics.org"><strong>Workflow4Metabolomics</strong></a> online
infrastructure jointly developed and maintained by the <a
href="http://www.france-bioinformatique.fr/en">French Institute of
Bioinformatics</a> and <a href="http://www.metabohub.fr/index.php?lang=en">MetaboHUB</a>; <a
href="http://dx.doi.org/10.1093/bioinformatics/btu813"><strong>Giacomoni et al, 2015</strong></a>; <a
href="http://dx.doi.org/10.1016/j.biocel.2017.07.002"><strong>Guitton et al, 2017</strong></a>).</p>

<br style="margin-bottom:240px;"/>

<p style="text-align:center"><img alt="W4M" src="images/res_w4m_panel.png"
width="384" height="193" /></p>

</div>
</div>

<div id="footer">
</div>

</div>
</body>
</html>
